cclient/api/examples/example_get_transactions_to_approve.c

```diff
@@ -7,8 +7,8 @@
 
 #include "cclient/api/examples/cclient_examples.h"
 
-static tryte_t const *const REF_HASH =
-    (tryte_t *)"VOPPIXFBHNNEFAAZJLLKTTFMMVJZPHQCTROSU99KYHYBGLJYMCPGIQGSUW9NQQ9TOVTEOKFDCAJLXA999";
+// static tryte_t const *const REF_HASH =
```

Comment:
Why not just remove them?

Reply:
this is an example that showing people how to use this API.